Core/Dlr/Ast/TypeBinaryExpression.cs

```diff
@@ -125,7 +125,9 @@ public sealed class TypeBinaryExpression : Expression {
         // Helper that is used when re-eval of LHS is safe.
         private Expression ByValParameterTypeEqual(ParameterExpression value) {
             Expression getType = Expression.Call(value, typeof(object).GetMethod("GetType"));
-            
+
+            // TODO: remoting information is obsolete with net5.0
```

Comment:
Nice catch, todo-ticket please to document it but I'm not sure we're going to be touching this anytime soon.